build-tools/package.json

```diff
@@ -42,28 +40,17 @@
 		"syncpack:deps:fix": "syncpack set-semver-ranges --config syncpack.config.cjs",
 		"syncpack:versions": "syncpack list-mismatches --config syncpack.config.cjs",
 		"syncpack:versions:fix": "syncpack fix-mismatches --config syncpack.config.cjs",
-		"test": "npm run test:mocha && npm run test:jest && npm run test:realsvc",
-		"test:bail": "npm run test:mocha:bail && npm run test:jest:bail",
+		"test": "npm run test:mocha",
+		"test:bail": "npm run test:mocha:bail",
 		"test:copyresults": "copyfiles --exclude \"**/node_modules/**\" \"**/nyc/**\" nyc",
 		"test:coverage": "c8 npm run test:report",
-		"test:jest": "assign-test-ports && lerna run test:jest --concurrency 4 --stream --no-bail --no-sort",
-		"test:jest:bail": "assign-test-ports && lerna run test:jest --concurrency 4 --stream",
-		"test:jest:report": "assign-test-ports && lerna run test:jest --concurrency 4 --stream --no-bail --no-sort -- -- --ci --reporters=default --reporters=jest-junit",
 		"test:mocha": "lerna run test:mocha --stream --no-bail --no-sort",
 		"test:mocha:bail": "lerna run test:mocha --stream",
 		"test:mocha:report": "lerna run test:mocha --stream --no-bail --no-sort -- -- --reporter xunit --reporter-option output=nyc/mocha-junit-report.xml",
-		"test:realsvc": "lerna run test:realsvc --stream --no-bail --no-sort",
-		"test:realsvc:report": "lerna run test:realsvc:report --stream --no-bail --no-sort",
-		"test:report": "npm run test:mocha:report && npm run test:jest:report && npm run test:realsvc:report",
+		"test:report": "npm run test:mocha:report",
 		"tsc": "lerna run tsc --stream",
 		"tsc:fast": "fluid-build --root .. -t tsc --buildTools",
```

Comment:
Might as well replace this with the new arguments.

Reply:
Done.